Library/ViewModels/RewardCellViewModelTests.swift

```diff
@@ -104,7 +104,7 @@ final class RewardCellViewModelTests: TestCase {
 
     self.vm.inputs.configureWith(project: .template,
                                  rewardOrBacking: .left(.template |> Reward.lens.remaining .~ 0))
-    self.cardViewBackgroundColor.assertValues([UIColor.white, UIColor.ksr_grey_100])
+    self.cardViewBackgroundColor.assertValues([UIColor.white, UIColor.white])
```

Comment:
yeah I think you can kill this output and test and just set the background color in `bindStyles` since it never changes.